Source/astcenc_weight_align.cpp

```diff
@@ -45,11 +45,22 @@
  */
 
 #include "astcenc_internal.h"
+#include "astcenc_vecmathlib.h"
 
 #include <stdio.h>
 #include <cassert>
 
-static const float angular_steppings[44] = {
+#if ASTCENC_SIMD_WIDTH <= 4
+    #define ANGULAR_STEPS 44
+#elif ASTCENC_SIMD_WIDTH == 8
+    // AVX code path loops over these tables 8 elements at a time,
+    // so make sure to have their size a multiple of 8.
+    #define ANGULAR_STEPS 48
+#else
+    #error Unknown SIMD width
+#endif
+
+static const float angular_steppings[ANGULAR_STEPS] = {
```

Comment:
This needs to be `alignas(ASTCENC_VECALIGN) static const ...`, to ensure aligned loads work.